packages/terra-tag/src/RollupTag.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from 'react';
+import { injectIntl } from 'react-intl';
+import { KEY_SPACE, KEY_RETURN } from 'keycode-js';
+import { v4 as uuidv4 } from 'uuid';
+import classNamesBind from 'classnames/bind';
+import PropTypes from 'prop-types';
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import styles from './Tag.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Determines if the Tag List is rolled up or not.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * Callback function triggered on click/key press of the roll-up tag
+   */
+  onSelectRollUp: PropTypes.func,
+  /**
+   * Number of tags that are rolled up.
+   */
+  rollupCount: PropTypes.number,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+};
+
+const RollUpTag = (props) => {
+  const {
+    isCollapsed, onSelectRollUp, intl, rollupCount,
+  } = props;
+  const rollUpTagRef = useRef();
+  const theme = React.useContext(ThemeContext);
+
+  const handleOnSelectRollUp = (event) => {
+    rollUpTagRef.current.setAttribute('data-terra-rollup-tag-show-focus-styles', 'true');
```

Comment:
Why do we need to set a data attribute here?

Reply:
<img width="743" alt="Screenshot 2023-10-06 at 9 29 04 PM" src="https://github.com/cerner/terra-core/assets/132886427/7f554b23-6c67-43c8-825a-315121e16d5a">
we have used the dotted border for the focused tag item. Css change